Description: Work around predefined "bool" on some architectures
 "bool" is predefined on some Debian/Ubuntu architectures, confusing gtk2hsC2hs.
 Explicitly undefine it to work around this.
Author: Colin Watson <cjwatson@ubuntu.com>
Forwarded: no

Index: gtksourceview2-0.13.1.1/gtksourceview2.cabal
===================================================================
--- gtksourceview2-0.13.1.1.orig/gtksourceview2.cabal	2014-12-21 19:22:03.582807243 +0100
+++ gtksourceview2-0.13.1.1/gtksourceview2.cabal	2014-12-21 19:22:27.175287331 +0100
@@ -84,3 +84,5 @@
         pkgconfig-depends: gtksourceview-2.0 >= 2.0.2
         x-Types-Hierarchy: hierarchy.list
         build-depends:     gtk >=0.13.0.0 && <0.14
+
+        cpp-options: -Ubool
